cmd/root.go

```diff
@@ -52,7 +52,20 @@ var (
 
 func init() {
 	versionString = strings.TrimSpace(versionString)
-	userAgent = "cloud-sql-proxy/" + versionString
+	userAgent = userAgentString()
+}
+
+func userAgentString() string {
+	ua := "cloud-sql-proxy/" + versionString
+	runtime, ok := os.LookupEnv("CLOUD_SQL_PROXY_RUNTIME")
+	if !ok {
+		return ua
+	}
+	rv, ok := os.LookupEnv("CLOUD_SQL_PROXY_RUNTIME_VERSION")
```

Comment:
Shall we just use `CLOUD_SQL_PROXY_RUNTIME` and let the runtime include version?